tabbycat/utils/tables.py

```diff
@@ -870,8 +870,6 @@ def add_debate_ballot_link_column(self, debates, show_ballot=False):
             for debate in debates:
                 if not debate.confirmed_ballot:
                     ballot_links_data.append("")
-                elif self.tournament.pref('teams_in_debate') == 'bp' and debate.round.is_break_round:
-                    ballot_links_data.append("")
```

Comment:
I'm also curious about why we feel the need to permit a scoresheet view for result-only consensus ballots, given that the only thing to know about them is the final result, which is already shown in the results table.